lib/v2/substack/templates/description.art

```diff
@@ -0,0 +1,30 @@
+{{link}}
```

Comment:
Don't put the `link` in the content. It should appear in `ctx.state.data.item.link` only.